Common Tools/DeviceRecognizer.swift

```diff
@@ -0,0 +1,181 @@
+            case "iPhone14,5": return "iPhone 13"
+            case "iPhone14,2": return "iPhone 13 Pro"
+            case "iPhone14,3": return "iPhone 13 Pro Max"
+            case "iPad2,1", "iPad2,2", "iPad2,3", "iPad2,4": return "iPad 2"
+            case "iPad3,1", "iPad3,2",
+                 "iPad3,3": return "iPad (3rd generation)"
+            case "iPad3,4", "iPad3,5",
+                 "iPad3,6": return "iPad (4th generation)"
+            case "iPad6,11", "iPad6,12": return "iPad (5th generation)"
+            case "iPad7,5", "iPad7,6": return "iPad (6th generation)"
+            case "iPad7,11", "iPad7,12": return "iPad (7th generation)"
+            case "iPad11,6", "iPad11,7": return "iPad (8th generation)"
+            case "iPad12,1", "iPad12,2": return "iPad (9th generation)"
+            case "iPad4,1", "iPad4,2", "iPad4,3": return "iPad Air"
+            case "iPad5,3", "iPad5,4": return "iPad Air 2"
+            case "iPad11,3", "iPad11,4": return "iPad Air (3rd generation)"
+            case "iPad13,1", "iPad13,2": return "iPad Air (4th generation)"
+            case "iPad2,5", "iPad2,6", "iPad2,7": return "iPad mini"
+            case "iPad4,4", "iPad4,5", "iPad4,6": return "iPad mini 2"
+            case "iPad4,7", "iPad4,8", "iPad4,9": return "iPad mini 3"
+            case "iPad5,1", "iPad5,2": return "iPad mini 4"
+            case "iPad11,1", "iPad11,2": return "iPad mini (5th generation)"
+            case "iPad14,1", "iPad14,2": return "iPad mini (6th generation)"
+            case "iPad6,3", "iPad6,4": return "iPad Pro (9.7-inch)"
+            case "iPad7,3", "iPad7,4": return "iPad Pro (10.5-inch)"
+            case "iPad8,1", "iPad8,2", "iPad8,3",
+                 "iPad8,4": return "iPad Pro (11-inch) (1st generation)"
+            case "iPad8,9",
+                 "iPad8,10": return "iPad Pro (11-inch) (2nd generation)"
+            case "iPad13,4", "iPad13,5", "iPad13,6",
+                 "iPad13,7": return "iPad Pro (11-inch) (3rd generation)"
+            case "iPad6,7",
+                 "iPad6,8": return "iPad Pro (12.9-inch) (1st generation)"
+            case "iPad7,1",
+                 "iPad7,2": return "iPad Pro (12.9-inch) (2nd generation)"
+            case "iPad8,5", "iPad8,6", "iPad8,7",
+                 "iPad8,8": return "iPad Pro (12.9-inch) (3rd generation)"
+            case "iPad8,11",
+                 "iPad8,12": return "iPad Pro (12.9-inch) (4th generation)"
+            case "iPad13,8", "iPad13,9", "iPad13,10",
+                 "iPad13,11": return "iPad Pro (12.9-inch) (5th generation)"
+            case "AppleTV5,3": return "Apple TV"
+            case "AppleTV6,2": return "Apple TV 4K"
+            case "AudioAccessory1,1": return "HomePod"
+            case "AudioAccessory5,1": return "HomePod mini"
+            case "i386",
+                 "x86_64": return "Simulator \(mapToDevice(identifier: ProcessInfo().environment["SIMULATOR_MODEL_IDENTIFIER"] ?? "iOS"))"
```

Comment:
Line Length Violation: Line should be 120 characters or less; currently it has 138 characters (line_length)